src/main/java/co/novu/api/integrations/IntegrationsApi.java

```diff
@@ -0,0 +1,38 @@
+package co.novu.api.integrations;
+
+import co.novu.api.integrations.requests.IntegrationRequest;
+import co.novu.api.integrations.responses.BulkIntegrationResponse;
+import co.novu.api.integrations.responses.ProviderWebhookStatusResponse;
+import co.novu.api.integrations.responses.SingleIntegrationResponse;
+import retrofit2.Call;
+import retrofit2.http.DELETE;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+import retrofit2.http.PUT;
+import retrofit2.http.Path;
+
+public interface IntegrationsApi {
+
+    String ENDPOINT = "integrations";
+    
+    @GET(ENDPOINT + "/get-integrations")
+    public Call<BulkIntegrationResponse> getIntegrations();
+
+    @POST(ENDPOINT + "/create-integrations/{request}")
+    public Call<SingleIntegrationResponse> createIntegration(@Path("request") IntegrationRequest request);
```

Comment:
The request cannot be specified as path. Please check out other merged PRs on how to handle this